experimental/examples/chat-mpp/iosApp/iosApp/ComposeViewControllerToSwiftUI.swift

```diff
@@ -0,0 +1,38 @@
+import UIKit
+import SwiftUI
+import shared
+
+struct ComposeViewControllerToSwiftUI: UIViewControllerRepresentable {
+    func makeUIViewController(context: Context) -> UIViewController {
+        return Main_iosKt.MainViewController()
+    }
+
+    func updateUIViewController(_ uiViewController: UIViewController, context: Context) {
+    }
+
+    func sizeThatFits(_ proposal: ProposedViewSize, uiViewController: UIViewControllerType, context: Context) -> CGSize? {
+        return uiViewController.view.frame.size
+    }
+}
+
+public func sendMessage(_ text: String) {
+    Main_iosKt.sendMessage(text: text)
+}
+
+public func gradient3Colors() -> [Color] {
+    return Main_iosKt.gradient3Colors().map { hex in
+        Color(getCGColor(hex.intValue)).opacity(1.0)
+    }
+}
+
+public func surfaceColor() -> Color {
+    Color(getCGColor(Int(Main_iosKt.surfaceColor())))
+}
+
+private func getCGColor(_ argb: Int) -> CGColor {
+    func clr(_ component: Int) -> CGFloat {
+        CGFloat(component & 0xff) / 255.0
+    }
+
+    return CGColor(red: clr(argb >> 16), green: clr(argb >> 8), blue: clr(argb), alpha: clr(argb >> 24))
+}
```

Comment:
Can you move these functions out into a separate file? They don't seem like they're actually related to the "ComposeViewControllerToSwiftUI" adapter :)

Reply:
Ok, sure. It is just sugar functions.

Reply:
Done